src/main/java/org/team1540/robot2019/utils/MiniPID.java

```diff
@@ -0,0 +1,461 @@
+package org.team1540.robot2019.utils;
+
+/**
```

Comment:
Something something plagarism but you gave credit so I guess it's good lol

Would be nice to pull this in as a JitPack dependency but the man didn't check in his hecking build files